### ISSUE TITLE
[FEATURE REQUEST] Add option to show total home load and load minus individual on card

### ISSUE THREAD

**Issue by obiphi** [flixlix/power-flow-card-plus#643]
I'd like it to be possible for the card to show the total home load AND the home load minus the individual loads, e.g. 

![image](https://github.com/flixlix/power-flow-card-plus/assets/32672400/2adcd215-7053-4f99-acde-61f18dd54268)

Would this be possible?

Thanks very much, the card is brilliant already!

**Comment by bjoernhardegen** [comment]
This is already an option in the home Configuration. It is called "substract individual"

**Comment by obiphi** [comment]
My apologies, I wasn't clear. I'd like to request that Both the 'total home load' And the 'total home load with individuals subtracted' be shown at the same time, hence my screenshot with two numbers in the Home box. Is that possible?

**Comment by bjoernhardegen** [comment]
That's the feature"substract individual". Try it.

**Comment by obiphi** [comment]
Thanks for your reply, but I'm already using it, see here's the card with subtract individual disabled:
![Screenshot_20240615_051329_Home Assistant](https://github.com/flixlix/power-flow-card-plus/assets/32672400/f4fd4fc2-4af4-4687-bc30-3c634174a70b)
And here it is with subtract individual enabled
![Screenshot_20240615_051345_Home Assistant](https://github.com/flixlix/power-flow-card-plus/assets/32672400/e80a21e6-51f6-4ea4-a0e0-f43f142d88ec)
I want to see both 400W and 2.5KW at the same time, both within the Home circle. Like the initial image on this post where I used Microsoft Paint to put 977W within the Home circle with the 479W provided by the card.

**Comment by bjoernhardegen** [comment]
Well I looked at my configuration and now I know why it isn't working for you.

I just added the home consumption as the secondary info again.  So basically I have the home power draw as the main entity, enabled the substraction to get 400W in your screenshots and then I configured the home power draw again as the secondary info. I must have forgotten that. Sorry. 

I currently have no individual item consuming energy but it looks like this. The top is the overall consumption und the button is the substracted value.

![Screenshot_20240615-064842](https://github.com/flixlix/power-flow-card-plus/assets/152603977/2accffce-8734-4501-b0d8-835600683380)
